src/components/Todo.js

```diff
@@ -1,19 +1,32 @@
 import PropTypes from 'prop-types'
 import Checkbox from '@material-ui/core/Checkbox'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Todos from './../todos'
 
-const Todo = ({ todo }) => {
+const Todo = ({ todo, notifyChange }) => {
     return (
         <li>
-            <Checkbox
-                checked={todo.completed}
+            <FormControlLabel
+                control={
+                    <Checkbox
+                        checked={todo.completed}
+                        onChange={(event) => {
+                            Todos.update(todo,
+                                {
+                                    "completed": event.target.checked
+                                }).then(() => notifyChange())
+                        }}
+                    />
+                }
+                label={todo.completed ? <s>{todo.title}</s> : todo.title}
```

Comment:
This doesn't look like a form, maybe we shouldn't use the FormControlLabel component here. But you should use this for the previous comment.